cmd/swarm/swarm-smoke/upload_and_sync.go

```diff
@@ -107,16 +138,82 @@ func cliUploadAndSync(c *cli.Context) error {
 
 // fetch is getting the requested `hash` from the `endpoint` and compares it with the `original` file
 func fetch(hash string, endpoint string, original []byte, ruid string) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	ctx, sp := spancontext.StartSpan(ctx, "upload-and-sync.fetch")
```

Comment:
I am not sure who is going to push this trace to the tracing system. It looks like `Swarm` might use it as a root span, but I don't think Swarm itself will push the full trace, I think this should happen wherever the root span is created.